### DISCUSSION TITLE
All the images of comments on this website cannot be saved.

### DISCUSSION THREAD

**Discussion by qixidao** [gildas-lormeau/SingleFile#1638] [Q&A]
**Describe the bug**
All the images of comments on this website cannot be saved.

**To Reproduce**
1.Go to 'https://www.52pojie.cn/thread-1986898-1-1.html' 
2.Save .
3.Check the result.

**Screenshots**
How the first link looks in browser:
![1](https://github.com/user-attachments/assets/95421bbc-ffba-427e-a61e-a807dceb81de)
How the saved version looks like:
![2](https://github.com/user-attachments/assets/56c72ef0-36a4-422d-9c2c-b8b25fab36eb)

**Environment**
 - OS: [ Win10 Pro]
 - Browser: [ Chrome]
 - Version: [e.g. 64]

**Comment by gildas-lormeau** [comment]
Thank you, I confirm I can reproduce the issue.

Unfortunately, this site has implemented various techniques to prevent [“hotlinking”](https://simple.wikipedia.org/wiki/Hotlinking) and this prevents SingleFile from downloading the images. I haven't done the test, but it's possible that installing an extension to remove CORS headers would help, e.g. https://chromewebstore.google.com/detail/cors-unblock/lfhmikememgdcahcdlaciloancbhjino.

**Comment by qixidao** [comment]
Thank you, this method is indeed effective。

**Reply by gildas-lormeau** [reply]
The version `1.22.78` should fix this issue. You should not need any additional extensions to circumvent hotlinking issues when using SingleFile in Chrome.
